hoot-core/src/main/cpp/hoot/core/schema/OsmSchema.h

```diff
@@ -68,7 +68,8 @@ struct OsmSchemaCategory
     PseudoName =      0x20,
     HgisPoi =         0x40, // Human Geography POI. See ticket #6853 for a definition of a "HGIS POI"
     Multiuse =        0x80,
-    All = Poi | Building | Transportation | Use | Name | HgisPoi | Multiuse
+    //TODO: should this have PseudoName too?
```

Comment:
I think you are correct, though `OsmSchemaCategory::Type::All` is never used anywhere in the code base.

Reply:
Interesting.  I can't think of a situation where you'd want to use "All".  I'm going to just comment it out for now.

Reply:
I'm also good with that.